etor/src/main/resources/transformation_definitions.json

```diff
@@ -181,8 +181,8 @@
           "name": "UpdateUniversalServiceIdentifier",
           "args": {
             "checkValue": "54089-8",
-            "codingSystem": "LN",
-            "alternateId": "CDPHGSPEAP"
+            "codingSystem": "CDPHGSPEAP",
+            "alternateId": ""
```

Comment:
I wasn't sure if this should remain or not because it was the same value, so I left it here and in the tests.

Reply:
It's only used on `UpdateUniversalServiceIdentifier`, so let's see if we can take it out safely.